tools/spec2vec/spec2vec_training.xml

```diff
@@ -0,0 +1,254 @@
+<tool id="spec2vec_training" name="Spec2Vec Model Training" version="@TOOL_VERSION@-@TOOL_DEV_VERSION@+galaxy0" python_template_version="3.5" profile="21.05">
+    <description>Train a Spec2Vec model for mass spectra similarity scoring</description>
+
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator"/>
+
+    <requirements>
+        <container type="docker">recetox/spec2vec:@COMMIT_SHA@</container>
+    </requirements>
+
+    <command detect_errors="exit_code"><![CDATA[
+        sh ${weights_symlink} &&
+        sh ${spec2vec_python_cli}
+    ]]></command>
+        
+    <configfiles>
+        <configfile name="weights_symlink">
+            ln -fs '${weights_filename}' '${weights_filename}.npy'
+        </configfile>
+        <configfile name="spec2vec_python_cli">
+            python3 ${__tool_directory__}/spec2vec_training_wrapper.py \
```

Comment:
single quotes are missing here

Reply:
This entire configfile is only a call to a python script that is on its own is call to a simple function.

If the python script was just created for this Galaxy tool you can avoid the argparse and the python script all together, you can put the function call directly into the `configfile` here. You essentially create the python script on the fly via Galaxy and execute it.

Reply:
I personally like the separation of having the individual wrapper file that handles the arguments and stuff on the python side related to the package and then having the galaxy related stuff in the tool script. I think the command line parsing might also at some point migrate into the spec2vec package to make that more user friendly, and then we can use it from here.

Reply:
Yes, I agree. If this is a separate package it is the preferred way.